MosaicSlider/InteractiveMosaic.js

```diff
@@ -21,7 +21,7 @@ export class InteractiveMosaic {
    * Constructor
    * @param {Color[]} colors An array of 4 colors to apply to the four quadrants of the mosaic grid
    */
-  constructor(colors) {
+  constructor(colors, sound) {
```

Comment:
I'm not a fan of passing in the sound class here since I'm not sure how I'll make a sound system long-term. Instead, have this class have an `events: EventTarget` that dispatches a `start-swap` event. the main sketch can subscribe to the event and trigger the sound